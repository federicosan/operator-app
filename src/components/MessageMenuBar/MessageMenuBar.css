.MessageMenuBar {
  align-items: stretch;
  box-sizing: border-box;
  height: 50px; /* TODO: Inherit the height of this component so it matches the siblings.*/
  justify-content: space-around;
  padding: 8px;
}

.MessageMenuBar .Button {
  height: 100%;
  line-height: inherit;
  font-weight: 400;
}
