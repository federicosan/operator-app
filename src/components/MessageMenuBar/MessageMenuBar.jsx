import React, { Component } from 'react';
import { connect } from 'react-redux';

// import PanelBar from '../PanelBar/PanelBar.jsx';

export class MessageMenuBarComponent extends Component {
  constructor (props) {
    super(props);

    this.state = {};
  }

  render () {
    return (
      <div className="row bar">
        <span>Menu</span>
      </div>
    );
  }
}

const mapStateToProps = state => ({

});

const mapDispatchToProps = dispatch => ({

});

const MessageMenuBar = connect(
  mapStateToProps,
  mapDispatchToProps,
)(MessageMenuBarComponent);

export default MessageMenuBar;
