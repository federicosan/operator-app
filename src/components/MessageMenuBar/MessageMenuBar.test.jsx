import React from 'react';

import { shallow } from 'enzyme';

import MessageMenuBar from './MessageMenuBar.jsx';

const store = {
  subscribe: jest.fn(),
  dispatch: jest.fn(),
  getState: jest.fn(() => ({ })),
};

describe('MessageMenuBar', () => {
  it('matches snapshot', () => {
    const component = shallow(<MessageMenuBar store={store} />);

    expect(component).toMatchSnapshot();
  });
});
