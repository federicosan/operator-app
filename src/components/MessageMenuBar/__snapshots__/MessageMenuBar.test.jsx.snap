exports[`MessageMenuBar matches snapshot 1`] = `
<MessageMenuBarComponent
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "subscribe": [Function],
    }
  } />
`;
